fuzz.go

```diff
@@ -1,6 +1,7 @@
+//go:build gofuzz
 // +build gofuzz
```

Comment:
BTW, now that we've switched to built-in `testing` fuzzer, maybe it's time to delete gofuzz support, and move its testcases to `FuzzUnserializer` seed input?